docs/standard/library-guidance/dependencies-net.md

```diff
@@ -0,0 +1,152 @@
+
+**Pros**
+- Balances compatibility with flexibility for fast-moving areas.
+- Encourages modernization and access to new features.
+- Simplifies decision-making and aligns with stability expectations.
+- Reduces complexity in dependency management.
+
+**Cons**
+- Increased infrastructure cost to maintain concurrent branches.
+- No new features for folks who want to stay on older dependencies.
+
+---
+
+## **Decision Matrix**
+
+| Strategy                              | Update Friction  | Engineering Cost |  Servicing Cost  |
+|---------------------------------------|------------------|------------------|------------------|
+| Latest Supported Versions (Option 1)  | Moderate         | Low              | Moderate         |
+| TFM-Specific Versions (Option 2)      | Low              | High             | Low / Moderate   |
+| Branching                             | Low              | Low              | High             |
+
+---
+
+## **Key Tradeoffs**
+
+- **Friction vs. Innovation:** Latest versions offer new features but have more friction for existing applications on older runtimes.
+- **Engineering cost:** Configuring multiple dependency groups, dependency updates for those, and managing API gaps can all accumulate to slow down innovation in a fast moving library.
+- **Servicing cost:** More package dependencies means more updates.  More branches mean more concurrent builds that need to stay healthy. Additional code in the form of polyfills means more LOC with potential bugs.
+
+---
+
+## **Recommended Guidance**
+
+- Option 1: Recommended for libraries moving fast and undergoing lots of innovation and feature development.  Consumers are more likely to be on the latest framework.
+- Option 2: Recommended for libraries that are stable and do not require new features.  Could be a transition strategy when a library reaches maturity and stops taking large features/
+- Option 3: Recommended for libraries that are part of .NET or receive strong customer feedback that option 1 is limiting consumption or blocking adoption.  Could be a transition strategy when Option 1 has too much friction.
+
+---
+
+## **Examples**
+
+### Example 1: Latest Supported Version
+```xml
+<PropertyGroup>
+  <TargetFrameworks>net8.0;net9.0;net10.0</TargetFrameworks>
+</PropertyGroup>
+<ItemGroup>
+  <PackageReference Include="System.Text.Json" Version="10.0.0" />
+</ItemGroup>
+```
+
+### Example 2: TFM-Specific Versions
+```xml
+<PropertyGroup>
+  <TargetFrameworks>net8.0;net9.0;net10.0</TargetFrameworks>
+</PropertyGroup>
+<ItemGroup>
+  <PackageReference Condition="'$(TargetFramework)' == 'net8.0'" Include="System.Text.Json" Version="8.0.0" />
```

Comment:
This example is a little odd because you don't need to PackageReference System.Text.Json at all here.

Reply:
Good point, I'll chose a different package and note the absence of STJ.